src/Feather.jl

```diff
@@ -282,44 +293,44 @@ function getmetadata{S,R}(io, T::Type{CategoricalValue{S,R}}, A)
 end
 
 values(A::Vector) = A
-values(A::NullableVector) = A.values
-values{S,R}(A::Union{CategoricalArray{S,1,R},NullableCategoricalArray{S,1,R}}) = map(x-> x - R(1), A.refs)
-values(A::DataArray) = A.data
+values(A::CategoricalArray{T,1,R}) where {T, R} = map(x-> x - R(1), A.refs)
```

Comment:
Just curious, where is it needed that values start at `0`? BTW, could be written as `A.refs - 1`, and `{T, R}` isn't needed (the array type is preserved).